armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -506,22 +506,30 @@ def _determineLinked(componentA, componentB):
         and isinstance(componentA, type(componentB))
         and (componentA.getDimension("mult") == componentB.getDimension("mult"))
     ):
-        idA, odA = (
-            componentA.getCircleInnerDiameter(cold=True),
-            componentA.getBoundingCircleOuterDiameter(cold=True),
-        )
-        idB, odB = (
-            componentB.getCircleInnerDiameter(cold=True),
-            componentB.getBoundingCircleOuterDiameter(cold=True),
-        )
+        try:
+            idA, odA = (
+                componentA.getCircleInnerDiameter(cold=True),
+                componentA.getBoundingCircleOuterDiameter(cold=True),
+            )
+            idB, odB = (
+                componentB.getCircleInnerDiameter(cold=True),
+                componentB.getBoundingCircleOuterDiameter(cold=True),
+            )
 
-        biggerID = max(idA, idB)
-        smallerOD = min(odA, odB)
-        if biggerID >= smallerOD:
-            # one object fits inside the other
+            biggerID = max(idA, idB)
+            smallerOD = min(odA, odB)
+            if biggerID >= smallerOD:
+                # one object fits inside the other
+                linked = False
+            else:
+                linked = True
+        except NotImplementedError:
+            runLog.warning(
+                "Components {0} and {1} do not have a 'getCircleInnerDiameter' and/or getBoundingCircleOuterDiameter method. "
+                "Instead of crashing and raising an error, I'm just going to assume they aren't linked.",
```

Comment:
Not a big deal at all, but my personal preference would be to avoid using first person in the warning.

Reply:
That's fair! And I don't think it's anywhere else in ARMI so I'll change it to be consistent with everything else. Though I think it's kind of fun when codes spit out first person error/warning messages 😄

Reply:
Addressed https://github.com/terrapower/armi/pull/778/commits/f7321919ca4c7f1b4dd47541d581407d51df6dcf